simtools/applications/make_regular_arrays.py

```diff
@@ -12,6 +12,10 @@
 
     Command line arguments
     ----------------------
+    site (str, required)
+        North or South.
+    model_version (str, optional)
+        Model version to use. If not provided, the latest version is used.
```

Comment:
I would still include an example here in the docstring, "e.g. prod5, prod6". That is simply because one might not understand immediately what model_version means

Reply:
OK, added `e.g., prod6`